src/ContainerManager.ts

```diff
@@ -5,7 +5,7 @@
  */
 import Component from "@egjs/component";
 import { DestroyOptions, SizeRect } from "./types";
-import { ResizeWatcher } from "./ResizeWatcher";
+import { OnResizeWatcherResize, ResizeWatcher } from "./ResizeWatcher";
```

Comment:
The word Resize seems to be repeated.
And what about camel case naming?
`onResizeWatcherObserve` or `onResizeWatcher`
https://vuejs.org/api/composition-api-lifecycle.html#onmounted

Reply:
Or just `ResizeEvent` or something.
You know, namings like `OnXXX` look more proper for methods(functions) or attributes.
For interfaces, I think using nouns/adjectives is more proper than using verbs.
Like, `Observer` / `Observable` but not `Observe`